src/Presentation/Console/Commands/InitSdkCommand.php

```diff
@@ -71,6 +72,31 @@ public function __construct(
         parent::__construct(static::NAME);
     }
 
+    /**
+     * @return void
+     */
+    protected function configure()
+    {
+        parent::configure();
+        $this->createDatabase();
+
+        $settings = $this->yamlParser->parseFile($this->settingsPath)['settings'];
```

Comment:
There is similar logic in InitProjectCommand. How do you think, is there a point in extracting it to abstract class or trait?